app/webpacker/stylesheets/components/pages/certification/_cs-accelerator.scss

```diff
@@ -11,8 +11,18 @@
   background-color: $concrete;
 }
 
-.ncce-programmes-activity__title {
-  color: unset;
+.ncce-progress-section {
+  display: flex;
+  flex: wrap;
+  flex-direction: column;
+
+  @include govuk-media-query($from: tablet) {
+    flex-direction: row;
+  }
+}
+
+.ncce-progress-section--text {
+  width: 200px;
 }
```

Comment:
At less than 770 px the row layout overflows beyond the right hand column and out of the viewport.
Consider using the column layout until you reach a breakpoint above 770px. 
<img width="771" alt="Screenshot 2023-07-26 at 12 24 46" src="https://github.com/NCCE/teachcomputing.org/assets/2813357/18d89b64-8ed1-433f-8a57-d78f6d98ece7">

Reply:
<img width="835" alt="Screenshot 2023-07-28 at 02 10 31" src="https://github.com/NCCE/teachcomputing.org/assets/14819641/bb1f1c0f-6ffd-44c8-9f07-75f61f8db1ba">

Reply:
This is still a (small) problem I think: at 770px it is OK, but at less than that it does this. (Firefox) 
<img width="799" alt="Screenshot 2023-08-02 at 09 54 07" src="https://github.com/NCCE/teachcomputing.org/assets/2813357/f6b2fdef-7baf-44f7-9b44-b5850b29017e">

(Google Chrome is slightly different but I think changing the breakpoint will also fix this.)
<img width="779" alt="Screenshot 2023-08-02 at 09 58 15" src="https://github.com/NCCE/teachcomputing.org/assets/2813357/fc1c89ff-6f52-471c-9f60-00356f0e9940">

Reply:
resolved thank you